lib/commands/generate.js

```diff
@@ -18,6 +18,7 @@ module.exports = Command.extend({
 
   availableOptions: [
     { name: 'dry-run',       type: Boolean, default: false, aliases: ['d'] },
+    { name: 'skip-prompts',  type: Boolean, default: false, aliases: ['sp'] },
```

Comment:
> @Turbo87 how would the --yes apply to multiple yes/no prompts?

that is exactly why I hadn't suggested `--yes` before. in this case something like `--use-defaults` probably makes more sense unless all the defaults are `yes` anyway.